src/components/search/SearchError.jsx

```diff
@@ -10,26 +10,24 @@ const SearchError = ({ title }) => {
   const renderDialog = useCallback(
     () => (
       <div className="lead d-flex align-items-center py-3">
```

Comment:
These CSS utility classes are messing with the styles (flexbox), and are no longer needed now that we're moving to an `Alert` that has built-in support for icons. We shouldn't need this wrapper `div` anymore.

Reply:
done